src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -123,6 +140,21 @@ public bool AutoReferenced
             }
         }
 
+        private string _installedVersionToolTip;
+
+        public string InstalledVersionToolTip
+        {
+            get
+            {
+                return _installedVersionToolTip;
+            }
+            set
+            {
+                _installedVersionToolTip = value;
+                OnPropertyChanged(nameof(InstalledVersionToolTip));
```

Comment:
nit/style question: Can't you omit the nameof and it'll pick it up with Reflection?